website/docs/.gitignore

```diff
@@ -7,6 +7,8 @@
 # Generated files
 .docusaurus
 .cache-loader
+docs/**/api
+sidebars
```

Comment:
why are `sidebars` ignored?

Reply:
The `docusaurus-plugin-typedoc` will create the sidebar files for you based on the output from typedoc. I needed a folder for the build to dump all of the sidebars into that wasn't part of the main docs and won't be commited.